Removed native date picker by making input tyle text

amir-qayyum-arbisoft opened the pull request:
Hi 
Previously app was showing 2 date pickers on chrome and one date picker on reset of browsers, I have changed input type=date to type=text, now we have one custom date picker on all browsers, 

@ahsan-ul-haq @muzaffaryousaf @symbolist

symbolist commented on lms/templates/ccx/schedule.html:
```diff
@@ -85,12 +85,20 @@
+        <!--
+          setting date input type to text, because if input type is date then chrome shows native date picker along
+          with custom date picker
+        --->
```
This type of comment should be added to the PR, not in the source. Please remove it.

symbolist commented on lms/templates/ccx/schedule.html:
```diff
@@ -85,12 +85,20 @@
+        <!--
+          setting date input type to text, because if input type is date then chrome shows native date picker along
+          with custom date picker
+        --->
```
Same here.

amir-qayyum-arbisoft replied:
Done @symbolist

amir-qayyum-arbisoft commented:
Added one validation,

*Added a validation, If user enters a date less then 100 , app with change it to **val + 2000** , means if user enters 0001-2-21 on focus out it will change it to 2001-2-21*

@ahsan-ul-haq @muzaffaryousaf @symbolist @afzaledx please take a look

symbolist commented on lms/templates/ccx/schedule.html:
```diff
@@ -124,5 +124,18 @@
     });
+    $('.datepair .date').change(function() {
+       var date = $(this).val();
+       var dateSplit;
```
Can you call this `dateParts`?

symbolist commented on lms/templates/ccx/schedule.html:
```diff
@@ -124,5 +124,18 @@
+       var date = $(this).val();
+       var dateSplit;
+       var year;
+       if (date){
```
Leave a space between `)` and `{`.

symbolist commented:
@amir-qayyum-arbisoft can this not do the job? https://jqueryui.com/datepicker/#min-max

Also you don't need to tag more than 2 people on a PR.

amir-qayyum-arbisoft commented:
No @symbolist the problem is when user enters year 0001 it show 2001 on calendar, I made things consistent by changing 0001 to 2001. Now on calendar and text field year is 2001
